src/tests/plugins.rs

```diff
@@ -91,16 +91,37 @@ fn heading_adapter_plugin() {
 fn syntect_plugin() {
     let adapter = crate::plugins::syntect::SyntectAdapter::new("base16-ocean.dark");
 
-    let input = concat!("```rust\n", "fn main<'a>();\n", "```\n");
-    let expected = concat!(
-        "<pre style=\"background-color:#2b303b;\"><code class=\"language-rust\">",
-        "<span style=\"color:#b48ead;\">fn </span><span style=\"color:#8fa1b3;\">main</span><span style=\"color:#c0c5ce;\">",
-        "&lt;</span><span style=\"color:#b48ead;\">&#39;a</span><span style=\"color:#c0c5ce;\">&gt;();\n</span>",
-        "</code></pre>\n"
-    );
+    let cases = vec![
+        (
+            concat!("```rust\n", "fn main<'a>();\n", "```\n"),
+            concat!(
+                "<pre style=\"background-color:#2b303b;\"><code class=\"language-rust\">",
+                "<span style=\"color:#b48ead;\">fn </span><span style=\"color:#8fa1b3;\">main</span><span style=\"color:#c0c5ce;\">",
+                "&lt;</span><span style=\"color:#b48ead;\">&#39;a</span><span style=\"color:#c0c5ce;\">&gt;();\n</span>",
+                "</code></pre>\n"
+            ),
+        ),
+        (
+            // Language should still be highlighted when delimited by a comma
+            "\
+            ```rust,ignore\n\
+            fn main() {}\n\
+            ```\n\
+            ",
+            "\
+            <pre style=\"background-color:#2b303b;\"><code class=\"language-rust,ignore\">\
```

Comment:
This still has `class="language-rust,ignore"`, so it wouldn't match anything looking for the class `language-rust`.

Reply:
I see now that this doesn't fit your use case. Changing that would differ from `cmark-gfm`'s output, but I should be able to add that functionality behind a parse option instead

I'll work on handling that today

Reply:
It does match GitHub's behavior though, so maybe `cmark-gfm` has a bug 🤔

Reply:
My guess is that GitHub uses the `full-info-string` option (which gives the full info string, not just the first word) and then parses the language out themselves. That's what we do.

Reply:
I'm not sure on how to proceed, so I'll just be awaiting more definitive feedback